site/cds_rdm/inspire_harvester/transform_entry.py

```diff
@@ -31,29 +32,44 @@ def _record(self):
 
     def _files(self):
         """Transformation of files."""
+        current_app.logger.info("Start transforming files.")
```

Comment:
personally I think this is too explicit. Imagine especially for 1000 records, each one of them will contain this message. Is it important?